configs/storage-proxy/sample.toml

```diff
@@ -167,6 +167,8 @@ path = "/vfroot/netapp"
 netapp_ontap_endpoint = "https://netapp.example.com"
 netapp_ontap_user = "admin-username"
 netapp_ontap_password = "admin-passwd"
+netapp_user_id = 1100   # Owner user id. Must be created on Netapp mgmt server.
+netapp_group_id = 1100  # Owner group id. Must be created on Netapp mgmt server.
```

Comment:
I think using the user and group IDs of the Storage Proxy should suffice. If the NetApp user and Storage Proxy user have different IDs, the Storage Proxy anyway cannot write files to the directory generated by Qtree, unless it is running with `root`.
https://github.com/lablup/backend.ai/blob/66c9e6c67d26958efa21b7c7ea8f7548cba445e0/configs/storage-proxy/sample.toml#L40-L41